tests/framework/common.go

```diff
@@ -1389,6 +1394,25 @@ func (td *OsmTestData) GetOsmCtlComponentRestarts() map[string]int {
 	return restartMap
 }
 
+// GetBugReport runs the "osm support bug-report" command
+func (td *OsmTestData) GetBugReport() error {
+	var args []string
+	args = append(args, "support",
+		"bug-report",
+	)
```

Comment:
```suggestion
	args := []string{"support", "bug-report"}
```

Reply:
In addition, the bug-report command must output the bug archive in the test specific directory determined by `td.GetTestDirPath()`. Use the `-o` option for this.